lego/apps/articles/fixtures/development_articles.yaml

```diff
@@ -15,16 +19,17 @@
   fields:
     title: Ny komité i Abakus
     cover: test_article_cover.png
-    description: Etter langt å lenge har HS endelig blitt enig.
-    text: Lorem Ipsum er rett og slett dummytekst fra og for trykkeindustrien.
-      Lorem Ipsum har vært bransjens standard for dummytekst helt siden 1500-tallet,
-      da en ukjent boktrykker stokket en mengde bokstaver for å lage et prøveeksemplar
-      av en bok. Lorem Ipsum har tålt tidens tann usedvanlig godt, og har i tillegg
-      til å bestå gjennom fem århundrer også tålt spranget over til elektronisk
-      typografi uten vesentlige endringer. Lorem Ipsum ble gjort allment kjent i
-      1960-årene ved lanseringen av Letraset-ark med avsnitt fra Lorem Ipsum, og
-      senere med sideombrekkingsprogrammet Aldus PageMaker som tok i bruk nettopp
-      Lorem Ipsum for dummytekst.
+    description: "Etter langt å lenge har HS endelig blitt enig."
+    text:
+      "<p>Lorem Ipsum er rett og slett dummytekst fra og for trykkeindustrien.\
+      \ Lorem Ipsum har vært bransjens standard for dummytekst helt siden 1500-tallet,\
+      \ da en ukjent boktrykker stokket en mengde bokstaver for å lage et prøveeksemplar\
+      \ av en bok. Lorem Ipsum har tålt tidens tann usedvanlig godt, og har i tillegg\
+      \ til å bestå gjennom fem århundrer også tålt spranget over til elektronisk\
+      \ typografi uten vesentlige endringer. Lorem Ipsum ble gjort allment kjent i\
+      \ 1960-årene ved lanseringen av Letraset-ark med avsnitt fra Lorem Ipsum, og\
+      \ senere med sideombrekkingsprogrammet Aldus PageMaker som tok i bruk nettopp\
+      \ Lorem Ipsum for dummytekst. </p>"
```

Comment:
Why all the backslashes? Seems a bit excessive to me :upside_down_face:

Reply:
Actually note sure, but that was how `<HTML>` was done in the other fixtures, so I just `cmd-c` + `cmd-v`